mods/noita-mp/files/scripts/util/NetworkUtils.lua

```diff
@@ -144,7 +141,7 @@ end
 --- @return boolean
 function NetworkUtils.alreadySent(peer, event, data)
     local cpc = CustomProfiler.start("NetworkUtils.alreadySent")
-
+    local clientCacheID = peer.clientCacheID or 0
```

Comment:
As mentioned previously, I don't like default values, as those lead to horrible bug I did it in the past. I was debugging for hours to find those bugs.

Reply:
In addition, think of test as mentioned below, to prevent strange behaviours.